tlslite/tlsconnection.py

```diff
@@ -1695,47 +1697,30 @@ def _serverCertKeyExchange(self, clientHello, serverHello,
         yield (premasterSecret, clientCertChain)
 
 
-    def _serverAnonKeyExchange(self, clientHello, serverHello, cipherSuite, 
-                               settings):
-        # Calculate DH p, g, Xs, Ys
-        # TODO make configurable
-        dh_g, dh_p = goodGroupParameters[2]
-        dh_Xs = bytesToNumber(getRandomBytes(32))
-        dh_Ys = powMod(dh_g, dh_Xs, dh_p)
+    def _serverAnonKeyExchange(self, serverHello, keyExchange, cipherSuite):
 
-        #Create ServerKeyExchange
-        serverKeyExchange = ServerKeyExchange(cipherSuite, self.version)
-        serverKeyExchange.createDH(dh_p, dh_g, dh_Ys)
-        
-        #Send ServerHello[, Certificate], ServerKeyExchange,
-        #ServerHelloDone  
+        # Create ServerKeyExchange
+        serverKeyExchange = keyExchange.makeServerKeyExchange()
+
+        # Send ServerHello[, Certificate], ServerKeyExchange,
+        # ServerHelloDone
         msgs = []
         msgs.append(serverHello)
         msgs.append(serverKeyExchange)
         msgs.append(ServerHelloDone())
         for result in self._sendMsgs(msgs):
             yield result
-        
-        #Get and check ClientKeyExchange
+
+        # Get and check ClientKeyExchange
         for result in self._getMsg(ContentType.handshake,
                                    HandshakeType.client_key_exchange,
                                    cipherSuite):
             if result in (0,1):
-                yield result 
+                yield result
             else:
                 break
         clientKeyExchange = result
-        dh_Yc = clientKeyExchange.dh_Yc
-        
-        if dh_Yc % dh_p == 0:
-            for result in self._sendError(AlertDescription.illegal_parameter,
-                    "Suspicious dh_Yc value"):
-                yield result
-            assert(False) # Just to ensure we don't fall through somehow            
-
-        #Calculate premaster secre
-        S = powMod(dh_Yc,dh_Xs,dh_p)
-        premasterSecret = numberToByteArray(S)
+        premasterSecret = keyExchange.processClientKeyExchange(clientKeyExchange)
```

Comment:
Yes, I know about it, but I´m not sure, how it should look like to be according to the PEP 8. 
Can you please write, how it should like? Where should be newline and how it should be indented?